### PR TITLE
Add airswap to `dex.trades_beta`

### PR DESCRIPTION
**Pull request by tomfutago**
linked to #4759

### CODE REVIEW

File: seeds/_sector/dex/uniswap_arbitrum_base_trades_seed.csv
Comment (Hosuke) [inline seeds/_sector/dex/uniswap_arbitrum_base_trades_seed.csv:1]: Do you want to add some seed records in this file?

File: seeds/_sector/dex/uniswap_bnb_base_trades_seed.csv
Comment (Hosuke) [inline seeds/_sector/dex/uniswap_bnb_base_trades_seed.csv:1]: And this file.

File: macros/models/_sector/dex/airswap_compatible_trades.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% macro airswap_compatible_trades(
+    blockchain = null,
+    project = null,
+    sources = null
```
Comment (Hosuke) [inline macros/models/_sector/dex/airswap_compatible_trades.sql:4]: Maybe we can make default sources as empty list to hint users.
```suggestion
    sources = []
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @tomfutago

File: macros/models/_sector/dex/airswap_compatible_trades.sql (hunk 1 above)
Comment (tomfutago) [reply macros/models/_sector/dex/airswap_compatible_trades.sql:4]: good idea!

File: models/_sector/dex/trades/arbitrum/_schema.yml [hunk 2]
```diff
@@ -15,5 +17,44 @@ models:
...
+    description: "Airswap arbitrum base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_seed:
+          seed_file: ref('airswap_arbitrum_base_trades_seed')
+          filter:
+            version: 3
```
Comment (Hosuke) [inline models/_sector/dex/trades/arbitrum/_schema.yml:51]: We may remove version filters here for project corresponding seed files.
```suggestion
```

File: models/_sector/dex/trades/bnb/_schema.yml [hunk 3]
```diff
@@ -15,5 +17,44 @@ models:
...
+    description: "Airswap bnb base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+      - check_seed:
+          seed_file: ref('airswap_bnb_base_trades_seed')
+          filter:
+            version: 3
```
Comment (Hosuke) [inline models/_sector/dex/trades/bnb/_schema.yml:51]: Same as above.
```suggestion
```

File: seeds/_sector/dex/airswap_arbitrum_base_trades_seed.csv [hunk 4]
```diff
@@ -0,0 +1,4 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+arbitrum,airswap,swap_erc20_v4,2023-07-06,0xdfe018b735bc8f15460d7dc2ce4d3d344c74182b6bdc8ae2cdb2a9b6414b2745,10,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,108480535,1000000000000000,10000
+arbitrum,airswap,swap_erc20_v4,2023-03-20,0xca7dfb88dc4bb6f39d5037498fab3778bc30dc667b2b9f25159d1d5b95a487da,5,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,0x82af49447d8a07e3bd95bd0d56f35241523fbab1,71897277,879419010851603072,500000000000000
```
Comment (Hosuke) [inline seeds/_sector/dex/airswap_arbitrum_base_trades_seed.csv:3]: As we have version values other than `3` here in the seed files.

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: seeds/_sector/dex/airswap_avalanche_c_base_trades_seed.csv [hunk 5]
```diff
@@ -0,0 +1,7 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+avalanche_c,airswap,swap,2023-07-17,0xd738f16bd636d9881fe17bc26ae860e87e72c57f8e7e4f95f63899c1e96a6760,13,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0xa7d7079b0fead91f3e65f86e8915cb59c1a4c664,32734320,94842568945986120,1340018
```
Comment (Hosuke) [inline seeds/_sector/dex/airswap_avalanche_c_base_trades_seed.csv:2]: https://snowtrace.io/tx/0xd738f16bd636d9881fe17bc26ae860e87e72c57f8e7e4f95f63899c1e96a6760

**Review by Hosuke — APPROVED** [review]
LGTM.✅ for the seeds

**Pull request merged**